plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--selector', type=str, action="store",
```

Comment:
I've adjusted the check to use a service name as input instead which is then used to lookup the selector(s) from the service definition and then those are passed into openshift.get_running_pod_names.  I also modified the function to support multi-pair selectors since the openshift service allows for it.

```
$ oc get service mongodb-service-1b -o json
{
    "kind": "Service",
    "apiVersion": "v1",
... 
   "spec": {
....
        "selector": {
            "foo": "bar",
            "name": "mongodb-replica"
        },
...
```